blackbox-tests/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: "3"
+services:
+  oplogtoredis:
+    build:
+      context: ..
+      dockerfile: blackbox-tests/Dockerfile.oplogtoredis-blackbox
+    command:
+      - /wait-for.sh
+      - mongo:27017
+      - '--'
+      - /wait-for.sh
+      - redis_tls:6380
+      - '--'
+      - sh
+      - -c
+      - |
+        mongo "$$OTR_MONGO_URL" --eval 'rs.initiate({ _id: "myapp", members: [{ _id: 0, host: "mongo:27017"}] })'
+        update-ca-certificates
+        oplogtoredis
+    environment:
+      - OTR_MONGO_URL=mongodb://mongo/dev
+      - OTR_REDIS_URL=redis://redis_tls:6380
+      - OTR_REDIS_TLS=true
+    ports:
+      - 9000:9000
+    depends_on:
+      - mongo
+      - redis_tls
+    volumes:
+      - ./certificates/ca.crt:/usr/local/share/ca-certificates/ca.crt:ro
+  mongo:
+    image: mongo:4.0.5
+    command: "mongod --replSet myapp --port 27017"
+    volumes:
+      - mongo_data:/data/db
+  redis:
```

Comment:
what is the point of this redis container? redis_tls below in referenced in OTR_REDIS_URL and the wait_for section, etc

Reply:
I've added a comment to clarify why we need both redis_tls and redis. I named it redis_tls because from a black box perspective it is redis_tls, but it really just contains stunnel.